explanations/maintainer-missing.md

```diff
@@ -0,0 +1,3 @@
+All packages ought to have a maintainer.
+
```

Comment:
Also need to add anchors to https://nixos.org/manual/nixpkgs/unstable/#sec-standard-meta-attributes so that we can link to the `maintainers` field docs.

Reply:
Opened https://github.com/NixOS/nixpkgs/pull/112331.